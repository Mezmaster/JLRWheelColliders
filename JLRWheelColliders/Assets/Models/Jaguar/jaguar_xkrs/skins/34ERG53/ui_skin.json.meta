fileFormatVersion: 2
guid: f4fbfa9dc13cac543803486f1f3dc850
TextScriptImporter:
  externalObjects: {}
  userData: 
  assetBundleName: 
  assetBundleVariant: 
