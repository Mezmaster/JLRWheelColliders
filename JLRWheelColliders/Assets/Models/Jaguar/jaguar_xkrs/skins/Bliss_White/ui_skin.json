{
    "skinname": "Bliss White",
    "drivername": "",
    "country": "",
    "team" : "",
    "number": 4
}