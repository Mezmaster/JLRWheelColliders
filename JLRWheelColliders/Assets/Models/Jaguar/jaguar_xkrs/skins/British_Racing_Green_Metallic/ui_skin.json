{
    "skinname": "British Racing Green Metallic",
    "drivername": "Sam Watson",
    "country": "uk",
    "team" : "British Racing Green Metallic",
    "number": 1
}