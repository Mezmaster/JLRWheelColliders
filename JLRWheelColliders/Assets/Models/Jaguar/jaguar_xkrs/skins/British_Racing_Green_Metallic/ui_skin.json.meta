fileFormatVersion: 2
guid: be832558f78498942b069989474dd70b
TextScriptImporter:
  externalObjects: {}
  userData: 
  assetBundleName: 
  assetBundleVariant: 
