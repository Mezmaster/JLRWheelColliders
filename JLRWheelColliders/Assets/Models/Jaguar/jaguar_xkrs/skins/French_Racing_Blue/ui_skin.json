{
    "skinname": "French Racing Blue",
    "drivername": "Simon Harrison",
    "country": "uk",
    "team" : "French Racing Blue",
    "number": 2
}