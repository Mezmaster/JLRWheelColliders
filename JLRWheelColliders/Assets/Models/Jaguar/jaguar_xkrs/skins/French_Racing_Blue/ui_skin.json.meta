fileFormatVersion: 2
guid: 1df2cef9e3fad9d4b9708a8fc902018b
TextScriptImporter:
  externalObjects: {}
  userData: 
  assetBundleName: 
  assetBundleVariant: 
