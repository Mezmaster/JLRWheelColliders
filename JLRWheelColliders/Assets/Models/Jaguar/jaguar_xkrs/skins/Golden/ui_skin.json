{
    "skinname": "Golden",
    "drivername": "Erg",
    "country": "uk",
    "team" : "",
    "number": 4
}